core/domain/prod_validators.py

```diff
@@ -1500,11 +1513,52 @@ def _validate_has_suggestion(cls, item):
                     'but a suggestion exists with id same as entity id' % (
                         item.id))
 
+    @classmethod
+    def _validate_original_author_id(cls, item):
+        """Validate that original author ID is in correct format.
+
+        Args:
+            item: GeneralFeedbackThreadModel. The model to validate.
+        """
+        if (
+                item.original_author_id and
+                not user_services.is_user_or_pseudonymous_id(
+                    item.original_author_id)
+        ):
+            cls._add_error(
+                'final %s' % (
+                    base_model_validators.ERROR_CATEGORY_AUTHOR_CHECK),
+                'Entity id %s: Original author ID %s is in a wrong format '
```

Comment:
The last sentence is run-on -- write two clear sentences instead.

Reply:
Done.